src/cpp/include/openvino/genai/continuous_batching_pipeline.hpp

```diff
@@ -30,7 +30,8 @@ class OPENVINO_GENAI_EXPORTS ContinuousBatchingPipeline {
     ContinuousBatchingPipeline(const std::string& models_path,
                                const SchedulerConfig& scheduler_config,
                                const std::string& device = "CPU",
-                               const ov::AnyMap& plugin_config = {});
+                               const ov::AnyMap& llm_plugin_config = {},
+                               const ov::AnyMap& tokenizer_plugin_config = {});
```

Comment:
@apaniukov just reminded. Please update python bindings

Reply:
You can also minimize the chance of us breaking `tokenizer_plugin_config` and Tokenizer's `plugin_config` args by adding tests. :) But since you are the only users of that feature it's up to you to write the tests.

Reply:
Added the bindings, rebased and resolved conflicts with latest master.
I did not add new tests, however I explicitly pass empty plugin config `{}` to Tokenizer/CB pipeline in tests, this way we should have ensured that nobody changes the constructor without modifying those tests/fixtures.

Please re-review @Wovchena